src/RollbarLogger.php

```diff
@@ -163,12 +177,12 @@ public function shouldIgnoreError($errno)
         return $this->config->shouldIgnoreError($errno);
     }
 
-    public function getQueueSize()
+    public function getQueueSize(): int
     {
         return count($this->queue);
     }
 
-    protected function send(\Rollbar\Payload\EncodedPayload $payload, $accessToken)
+    protected function send(EncodedPayload $payload, $accessToken): Response
```

Comment:
If I've read correctly, `string $accessToken` isn't allowed until PHP 7+, and that's why it's not used here and in the other function signatures that have a string argument?

Reply:
Good catch, @waltjones! I had those changes stashed and forgot to pop them before I made this PR. I've addressed them in an addendum PR #524.